src/main/java/de/tum/in/www1/artemis/service/plagiarism/ProgrammingPlagiarismDetectionService.java

```diff
@@ -137,7 +132,6 @@ public TextPlagiarismResult checkPlagiarism(long programmingExerciseId, float si
 
             log.info("JPlag programming comparison done in {}", TimeLogUtil.formatDurationFrom(start));
             plagiarismWebsocketService.notifyInstructorAboutPlagiarismState(topic, PlagiarismCheckState.COMPLETED, List.of());
-            limitAndSavePlagiarismResult(textPlagiarismResult);
```

Comment:
I don't understand why plagiarism results don't get stored at all anymore. Could you tell my why that is the case?

Reply:
Artemis limits the number of stored plagiarism results to 100. Number of results grows exponentially, so for large course it could be _huge_. In this PR we allow admins to configure the limit